tensorboard/plugins/histogram/summary.py

```diff
@@ -138,7 +138,7 @@ def op(name,
       display_name=display_name, description=description)
   with tf.name_scope(name):
     tensor = _buckets(data, bucket_count=bucket_count)
-    return tf.compat.v1.summary.tensor_summary(name='histogram_summary',
```

Comment:
Why change `tf.compat.v1.summary.tensor_summary` to `tf.summary.tensor_summary` here, but not also in (say) `event_accumulator_test`?

I note that `tf.compat.v1.summary.tensor_summary` still exists in the TF 2.0 preview. Are the symbols equivalent? If so, do you know why the converter script made this change in the first place? (Same question about `assert_scalar` below.)

Reply:
Resolved: <https://github.com/tensorflow/tensorboard/pull/1706#discussion_r243368145>